src/assets/release-notes-1.16.json

```
@@ -1,4 +1,50 @@
 {
+  "81068": {
+    "commit": "e7ecb2240375ac2b91f7bbd55358cded0eb1f698",
+    "text": "",
+    "markdown": "EvenPodsSpread feature has been added in alpha. This feature introduced an API field `pod.spec.topologySpreadConstraints` to enable users control how Pods are spread across the cluster among failure-domains.",
```

My suggestion for wording:
```suggestion
    "markdown": "Pod spread constraints have been added in alpha. You can use these constraints to control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))",
```
```
(also made in https://github.com/kubernetes/website/pull/16446)

---

Changed as requested :+1: